### ISSUE TITLE
Note type: TODO

### ISSUE THREAD

**Issue by orthecreedence** [turtl/project-tracker#42]
Has checkboxes, due dates, etc

Might just build on the markdown checkbox support? Would probably want core support.

> Blocked by #27

**Comment by snaggen** [comment]
I agree that core support is the way to go. Building it on markdown would probably give a less flexible solution. I suggest you take a look on how Google Keep handle checkbox lists, it is the best I found so far. 
Things that I think are a musts and nice to have for TODO-lists: 
MUST: Wysiwyg! You get a checkbox automatically at the beginning of the lines, and you can just click them to check them. 
MUST: The preview should also show the checkboxes. Not like Evernote where you only see the text in the preview, and have to open the note to see the actual list with checkboxes. 
NICE: When they are checked, they are greyed out and strikethrough to show that you are done with them.
NICE: Finished items are sorted at the bottom of the list, so the active TODOs are all easy to access at the top.

**Comment by dayfuaim** [comment]
MUST: You (un)check checkbox on preview – it have to be (un)checked and reflected in note text.

**Comment by clintlombard** [comment]
This is the one feature that's stopping me from leaving Google Keep completely
